<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserLoginWdg</class>
 <widget class="QWidget" name="UserLoginWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserLoginDlg</string>
  </property>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>380</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Resources/login/login_bg2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>25</y>
     <width>242</width>
     <height>46</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="countBtn">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>85</y>
     <width>221</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>账号登录</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="meetingBtn">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>85</y>
     <width>221</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>会议号登录</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="setBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>86</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>186</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>15</y>
     <width>17</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="saveCheckBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>保存密码</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="freeCheckBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>免费体验</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/Default.qrc"/>
 </resources>
 <connections/>
</ui>
